partcad/src/partcad/project.py

```
@@ -1061,10 +1061,11 @@ def get_provider(self, provider_name, func_params=None) -> provider.Provider:
 
             return self.providers[result_name]
 
-    def get_suppliers(self, part_spec):
-        # TODO(clairbee): return the eligible subset of suppliers
-        # part_name = part_spec["name"]
-        return self.suppliers
+    def get_suppliers(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_suppliers`

(ANN201)